spring-integration-file/src/main/java/org/springframework/integration/file/config/FileListFilterFactoryBean.java

```diff
@@ -54,6 +62,16 @@ public void setFilenameRegex(String filenameRegex) {
 		this.filenameRegex = filenameRegex;
 	}
 
+	/**
+	 * Specifies whether hidden files shall be ignored. This is {@code true}
+	 * by default.
+	 *
+	 * @param ignoreHidden Can be null, which triggers default behavior
+	 */
```

Comment:
`@since 4.2` and no one empty line according to Spring Framework JavaDoc rules.
Will be fixed on merge